examples/ts/btc/omni/index.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Send or create an omni asset from a multi-sig wallet at BitGo.
+ *
+ * Copyright 2024, BitGo, Inc.  All Rights Reserved.
+ */
+import { Wallet } from 'modules/bitgo/src';
+import { omniConfig } from './config';
+import * as superagent from 'superagent';
+import * as utxolib from '@bitgo/utxo-lib';
+
+const RECEIVE_ADDRESS = '';
+const SEND_ADDRESS = '';
+const AMOUNT = 729100000n;
+const ASSET_ID = 31;
+
+async function getWallet() {
+  return await omniConfig.sdk.coin(omniConfig.coin).wallets().get({ id: omniConfig.walletId });
+}
+
+async function sendOmniAsset(
```

Comment:
I would add a pretty verbose jsdoc ontop here to explain

1. the situations that you would use this (omni was sent to one of the addresses on the bitgo btc wallet
2. What this assumes (assumes that you have an unspent at the address for example)
3. What each parameter means, and maybe a link that the user can find it if necessary. For example - `assetId` you can have a link to the assets. For `omniAmountMicroCents`, have a link to the explorer where they can find that amount